docs/00-getting-started.md

```diff
@@ -53,15 +59,17 @@ Once you have one, simply log in and go to the [Azure Portal](https://portal.azu
 
 4. Save your account credentials into the secrets.yaml file
 
-- Copy the secrets.yaml.template file to secrests.cfg
+- Copy the secrets.yaml.template file in the .thunderbolt/ directory to .thunderbolt/secrets.yaml
 
 Windows
 ```sh
+cd .thunderbolt
 copy secrets.yaml.template secrets.yaml
 ```
 
 Linux and Mac OS
 ```sh
+cd .thunderbolt
 cp secrets.yaml.template secrets.yaml
```

Comment:
cp? it should just give you secrets.yaml and not secrets.yaml.template

Reply:
agreed. If we're going down the 'init' path then we should just create the secrets.yaml file. The template file will be distributed with our package (I suppose) and copied into the .thunderbolt on the init call.

Reply:
yeah, the latest commit fixed this already. Now `azb spark init` automatically renames the secrets file for you. These docs should be updated to reflect that.